mne/beamformer/_lcmv.py

```diff
@@ -285,6 +353,12 @@ def lcmv(evoked, forward, noise_cov, data_cov, reg=0.05, label=None,
         detected automatically. If int, the rank is specified for the MEG
         channels. A dictionary with entries 'eeg' and/or 'meg' can be used
         to specify the rank for each modality.
+    apply_noise_norm: False | True
+        If True, a noise-normalized beamformer will be computed.
```

Comment:
Short answer: I don't know :-) -- see initial commit. 
I made the already implemented noise normalization optional to make a comparison to Fieldtrip's output possible. But whether they make sense next to each other or in combination or if we want to merge them into one "normalization" option ...
cc @sarangnemo